packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -278,6 +279,54 @@ class Opacity extends SingleChildRenderObjectWidget {
   /// would otherwise contribute relevant semantics.
   final bool alwaysIncludeSemantics;
 
+  /// Whether to wrap the provided child widget in a [RepaintBoundary].
+  ///
+  /// This value defaults to true. The repaint boundary allows the engine
+  /// to reuse textures on subsequent frames if the opacity is changing but
+  /// the child layer is not. If the child layers are usually changing along
```

Comment:
See also my note here: https://github.com/flutter/flutter/issues/101597#issuecomment-1093404620

If we decide to keep it on Opacity, we may want to link to AnimatedOpacity and FadeTransition from here and emphasize that those may be better choices if the opacity changes frequently?

Reply:
The fact that we need a switch to turn this off makes me wonder if we should actually just omit this feature from the Opacity widget. Did you see a use case where this helped? I would have assumed that this is mostly helpful for animated opacities (and for those one should use AnimatedOpacity or FadeTransition).

Reply:
So this optimiztion isn't really about animated opacities, its just easier to see when the opacity is animated. Anytime you repaint anything above an opacity, whether or not the opacity or its children have changed - you will go down a much slower path if the engine thinks the child layers have been updated.

Reply:
In this particular case though, I'm not really in love with this flag. I think we should just update the progress spinner to not use an opacity widget and instead paint with opacity.

Reply:
> I think we should just update the progress spinner to not use an opacity widget and instead paint with opacity.

+1 to that if that's possible.